src/opencensus/proto/metrics/v1/metrics.proto

```diff
@@ -151,10 +159,8 @@ message Point {
     // A distribution value.
     DistributionValue distribution_value = 4;
 
-    // TODO: Add support for Summary type. This is an aggregation that produces
-    // percentiles directly.
-    //
-    // See also: https://prometheus.io/docs/concepts/metric_types/#summary
+    // A summary value.
```

Comment:
Maybe worth repeating here that it's not recommended.

Reply:
done.